packages/ui/src/pages/Home/MultisigView.tsx

```diff
@@ -1,61 +1,89 @@
 import AccountDisplay from '../../components/AccountDisplay'
 import { AccountBadge } from '../../types'
-import { renderMultisigHeading } from '../multisigHelpers'
 import MultisigActionMenu from './MultisigActionMenu'
 import { styled } from '@mui/material/styles'
 import { Chip, Paper } from '@mui/material'
 import { useMultiProxy } from '../../contexts/MultiProxyContext'
+import Accordion from '../../components/library/Accordion'
+import { Button } from '../../components/library'
+import Balance from '../../components/library/Balance'
 
 const MultisigView = () => {
   const { selectedMultiProxy, selectedHasProxy, selectedIsWatched } = useMultiProxy()
-
   return (
-    <>
-      <HeaderStyled>
-        {selectedMultiProxy && (
-          <h3>{renderMultisigHeading(selectedMultiProxy.multisigs.length > 1)}</h3>
-        )}
-        <BoxStyled>{!selectedIsWatched && !selectedHasProxy && <MultisigActionMenu />}</BoxStyled>
-      </HeaderStyled>
+    <MultisigWrapperStyled>
+      <h3>Controlled by</h3>
       {selectedMultiProxy &&
         selectedMultiProxy.multisigs.map((multisig) => {
           return (
-            <MultisigPaperStyled key={multisig.address}>
-              <AccountDisplayWrapperStyled>
-                <AccountDisplay
-                  address={multisig.address || ''}
-                  badge={AccountBadge.MULTI}
-                  withBalance
-                />
-              </AccountDisplayWrapperStyled>
-              <SignatoriesWrapperStyled>
-                <h4>
-                  Signatories{' '}
+            <Accordion
+              header={
+                <AccountDisplayWrapperStyled>
+                  <AccountDisplay
+                    address={multisig.address || ''}
+                    badge={AccountBadge.MULTI}
+                    withBalance={false}
```

Comment:
I really think we should keep the balance visible at all time, and keep this.
![image](https://github.com/ChainSafe/Multix/assets/33178835/6f13ef9e-b733-4ea8-85e4-80d592e5d962)

Reply:
Answering here regarding your comment:
> Always show a balance on multisig (imo it stretch the multisig selector too much that it looks disproportionately )

Agreed, maybe we should put it right to the Multisig, in its own box like with do with the header? But my point remains, we should see the balance without an additional click. Also I find it not nice right now in it own box below, it looks out of place.

@sweetpea22 help :D